lambdas/tests/unit/handlers/test_generate_lloyd_george_stitch_handler.py

```diff
@@ -0,0 +1,221 @@
+    actual = lambda_handler({}, context)
+    expected = ApiGatewayResponse(
+        400, LambdaError.StitchClient.create_error_body(), "GET"
+    ).create_api_gateway_response()
+
+    assert expected == actual
+
+
+def test_400_response_thrown_if_no_new_trace_in_image(context, set_env):
+    actual = lambda_handler(INVALID_EVENT_EXAMPLE, context)
+    expected = ApiGatewayResponse(
+        400, LambdaError.StitchClient.create_error_body(), "GET"
+    ).create_api_gateway_response()
+
+    assert expected == actual
+
+
+def test_400_response_if_event_name_not_insert(context, set_env):
+    actual = lambda_handler(MODIFY_EVENT_EXAMPLE, context)
+    expected = ApiGatewayResponse(
+        400, LambdaError.StitchClient.create_error_body(), "GET"
+    ).create_api_gateway_response()
+
+    assert expected == actual
+
+
+def test_400_response_if_new_image_is_not_dictionary(context, set_env):
+    actual = lambda_handler(NEW_IMAGE_NOT_DICT_EXAMPLE, context)
+    expected = ApiGatewayResponse(
+        400, LambdaError.StitchClient.create_error_body(), "GET"
+    ).create_api_gateway_response()
+
+    assert expected == actual
+
+
+def test_handler_return_500_response_when_stitch_record_error(context, set_env, mocker):
+    mocker.patch(
+        "handlers.generate_lloyd_george_stitch_handler.stitch_record_handler",
+        side_effect=LGStitchServiceException(500, LambdaError.MockError),
+    )
+    mocker.patch(
+        "handlers.generate_lloyd_george_stitch_handler.prepare_stitch_trace_data"
+    )
+
+    actual = lambda_handler(MOCK_EVENT_RESPONSE, context)
+
+    expected = ApiGatewayResponse(
+        500, LambdaError.MockError.create_error_body(), "GET"
+    ).create_api_gateway_response()
+
+    assert expected == actual
+
+
+def test_stitch_record_handler_raise_error_if_zip_trace_model_validation_fails(
+    mock_lloyd_george_generate_stitch_service,
+):
+    with pytest.raises(LGStitchServiceException):
+        stitch_record_handler(INVALID_IMAGE)
+
+    mock_lloyd_george_generate_stitch_service.assert_not_called()
+
+
+def test_stitch_record_handler_happy_path(
+    mock_lloyd_george_generate_stitch_service,
+):
+    stitch_record_handler(PROCESSES_VALID_IMAGE)
+
+    mock_lloyd_george_generate_stitch_service.handle_stitch_request.assert_called_once()
+
+
+def test_zip_service_handle_stitch_request_called(
+    mock_lloyd_george_generate_stitch_service,
+):
+    mock_lloyd_george_generate_stitch_service.handle_stitch_request.side_effect = (
+        LGStitchServiceException("test", LambdaError.MockError)
+    )
+
+    with pytest.raises(LGStitchServiceException):
+        stitch_record_handler(PROCESSES_VALID_IMAGE)
+
+    mock_lloyd_george_generate_stitch_service.handle_stitch_request.assert_called_once()
+
+
+def test_prepare_stitch_trace_data():
+    actual = prepare_stitch_trace_data(VALID_NEW_IMAGE)
+    expected = PROCESSES_VALID_IMAGE
+
+    assert actual == expected
```

Comment:
Can we flip this to `expected == actual` to be consistent with the rest of the file?